### DISCUSSION TITLE
How to convert flux multiple piped files into a single SafeTensor file?

### DISCUSSION THREAD

**Discussion by kigy1** [huggingface/diffusers#9232] [Q&A]
hello everyone.....

I want to convert flux multiple file into a single safetensors file.

here there are the two kind of the file the multi and single but how I can make it one file like this
https://huggingface.co/black-forest-labs/FLUX.1-dev/tree/main

I already search [here](https://github.com/huggingface/diffusers/tree/main/scripts) but there is no script for that

thanks....

**Comment by kigy1** [comment]
@sayakpaul I thinks your are the hero

I would also ask [tokenizer](https://huggingface.co/black-forest-labs/FLUX.1-dev/tree/main/tokenizer) and [tokenizer_2](https://huggingface.co/black-forest-labs/FLUX.1-dev/tree/main/tokenizer_2)  is merged with which file ? , flux1-dev.safetensors or t5xxl_fp16.safetensors or clip_l.safetensors ???

Thank you, if my question is not clear please let me know to clarify

**Comment by DarkAlchy** [comment]
I still want to know how because I have a flux1-dev in diffusers that is not from BFL and need it in a .safetensors so I can train it in Kohya.

**Comment by PineREN** [comment]
Hello, may I ask if you have solved it? I also encountered the same problem. I fine-tuned the FLUX model, but I have no idea how to convert the diffusion models to a single.
